code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -85,14 +86,13 @@ func (edb *EventDb) GetTransactionByBlockNumbers(blockStart, blockEnd int, limit
 	tr := []Transaction{}
 	res := edb.Store.Get().
 		Model(Transaction{}).
-		Joins("INNER JOIN blocks on blocks.round >= ? AND blocks.round <= ? AND blocks.hash = transactions.block_hash", blockStart, blockEnd).
+		Where("round >= ? AND round <= ?", blockStart, blockEnd).
```

Comment:
why join was bad here?

Reply:
We just added a round field to the transaction table, we don't need to get it from another table anymore. I assume using an inner join to add a field that is in the original table takes longer.